src/NUnitFramework/tests/Internal/TextMessageWriterTests.cs

```diff
@@ -41,6 +41,23 @@ public void SetUp()
         }
 
         [Test]
+        public void TestMaLineLength()
```

Comment:
This method has a spelling mistake and you are testing many different (but related) things in this one test. I would prefer if it was broken up into discrete tests. I would also prefer if you saved off the current `DefaultLineLength` in `SetUp` and restore it in `TearDown` so that it has no chance of polluting other tests.